src/site/content/en/media/cheetsheet/index.md

```diff
@@ -0,0 +1,457 @@
+---
+layout: post
+title: Media Manipulation Cheat Sheet
+authors:
+  - joemedley
+description: |
+  An ordered rundown of commands needed to get from a raw mov file to encrypted
+  assets packaged for DASH or HLS.
+date: 2018-09-20
+updated: 2020-08-20
+---
+
+This page offers kinds sets of resources:
+
+* Commands for manipulating specific characteristics of media files.
+* The sequence of commands needed to get from a raw mov file to encrypted media assets.
+
+Conversion is done with these applications:
+
+* [Shaka Packager](https://github.com/google/shaka-packager) ([on StackOverflow](https://stackoverflow.com/questions/tagged/shaka))
+* [ffmpeg](https://ffmpeg.org/download.html), version 4.2.2-tessus ([on StackOverflow](https://stackoverflow.com/questions/tagged/ffmpeg))
+* [OpenSSL](https://www.openssl.org/)  ([on StackOverflow](https://stackoverflow.com/questions/tagged/openssl))
+
+
+Although I've tried to show equivalent operations for all procedures, not all
+operations are possible in both applications.
+
+In many cases, the commands I'm showing may be combined in a single command line
+operation, and in actual use would be. For example, there's nothing preventing
+you from setting an output file's bitrate in the same operation as a file
+conversion. For this cheat sheet, I often show these operations as separate
+commands for the sake of clarity.
+
+Please let me know of useful additions or corrections.
+[Pull requests are welcome](https://github.com/GoogleChrome/web.dev/tree/media/src/site/content/en/media/cheetsheet).
+
```

Comment:
Can we also say something like "don't worry if you don't understand what these things mean right now"